rmgpy/data/kinetics/family.py

```diff
@@ -1752,9 +1752,11 @@ def __generateReactions(self, reactants, products=None, forward=True, prod_reson
                         # Iterate over each pair of matches (A, B)
                         for mapA in mappingsA:
                             for mapB in mappingsB:
-                                reactantStructures = [moleculeA, moleculeB]
+                                # Reverse the order of reactants in case we have a family with only one reactant tree
```

Comment:
If I understand this correctly this should probably be squashed in with some of the earlier stuff

Reply:
Done